crs_motion_planning/src/path_planning_utils.cpp

```diff
@@ -109,300 +109,389 @@ bool crsMotionPlanner::generateSurfacePlans(pathPlanningResults::Ptr& results)
       std::vector<geometry_msgs::msg::PoseArray> split_rasters;
       // Split up raster based on where planning failures occurred
       geometry_msgs::msg::PoseArray curr_failed_vertex_poses;
-      std::cout << "CLEANING" << std::endl;
+      std::cout << "SOME POINTS UNREACHABLE, CLEANING..." << std::endl;
       crs_motion_planning::cleanRasterStrip(strip, failed_vertices, split_rasters, curr_failed_vertex_poses);
 
-      // Display failed vertices
+      // Store failed vertices
       results->unreachable_waypoints.poses.insert(results->unreachable_waypoints.poses.end(),
                                                   curr_failed_vertex_poses.poses.begin(),
                                                   curr_failed_vertex_poses.poses.end());
 
       for (auto split_strip : split_rasters)
       {
-        // Generate Descartes preplan
         if (split_strip.poses.size() >= config_->minimum_raster_length)
         {
-          geometry_msgs::msg::PoseArray split_strip_ar;
-
-          if (config_->add_approach_and_retreat)
-          {
-            crs_motion_planning::addApproachAndRetreat(
-                split_strip, config_->approach_distance, config_->retreat_distance, split_strip_ar);
-          }
-          else
-          {
-            split_strip_ar = split_strip;
-          }
-          std::cout << "DESCARTES ROUND 2" << std::endl;
           results->reachable_waypoints.poses.insert(
               results->reachable_waypoints.poses.end(), split_strip.poses.begin(), split_strip.poses.end());
-          if (generateDescartesSeed(split_strip_ar, failed_edges, failed_vertices, joint_traj_msg_out_final))
-          {
-            split_traj.push_back(joint_traj_msg_out_final);
-            split_reachable_rasters.push_back(std::move(split_strip_ar));
-            any_successes = true;
-            std::cout << "SUCCESS" << std::endl;
-          }
-          else
-          {
-            results->unreachable_waypoints.poses.insert(
-                results->unreachable_waypoints.poses.end(), split_strip_ar.poses.begin(), split_strip_ar.poses.end());
-            std::cout << "ADDED APPROACH AND/OR RETREAT MADE RASTER UNREACHABLE" << std::endl;
-          }
+          split_reachable_rasters.push_back(std::move(split_strip));
+          any_successes = true;
         }
         else
         {
           results->skipped_rasters.push_back(split_strip);
         }
-        std::cout << "DONE" << std::endl;
       }
+      std::cout << "DONE" << std::endl;
```

Comment:
I'd prefer it if we used `RCLCPP_ ... ` macros a those can be toggled on/off from outside the node

Reply:
You could use the [rclcpp::get_logger(...) function ](http://docs.ros2.org/bouncy/api/rclcpp/namespacerclcpp.html#ae7295751947c08312aa69f45fd673171) to get an instance of a logger

Reply:
I am making these changes on #37